src/main/java/org/localhost/pizzeria/messaging/config/RabbitMQConfig.java

```diff
@@ -1,30 +1,36 @@
 package org.localhost.pizzeria.messaging.config;
 
-import lombok.Value;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.localhost.pizzeria.messaging.consumer.SupplyRestockConsumer;
+import org.springframework.amqp.rabbit.connection.ConnectionFactory;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.amqp.support.converter.Jackson2JsonMessageConverter;
+import org.springframework.amqp.support.converter.MessageConverter;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
 
-@Value
+@Configuration
 public class RabbitMQConfig {
-    String host;
-    int port;
-    String username;
-    String password;
-    String supplyCheckQueue;
-    String supplyCheckExchange;
-    String supplyCheckRoutingKey;
-    String dlqQueue;
-    String dlqExchange;
 
-    public static RabbitMQConfig getDefaultConfig() {
-        return new RabbitMQConfig(
-                "localhost",
-                5672,
-                "guest",
-                "guest",
-                "supply-check-queue",
-                "supply-check-exchange",
-                "supply-check-routing-key",
-                "supply-check-dlq",
-                "supply-check-dlq-exchange"
-        );
+    @Bean
+    public ObjectMapper objectMapper() {
+        return new ObjectMapper();
+    }
+
+    @Bean
+    public MessageConverter jsonMessageConverter() {
+        return new Jackson2JsonMessageConverter();
+    }
```

Comment:
@Bean
    public ObjectMapper objectMapper() {
        return new ObjectMapper();
    }

    @Bean
    public MessageConverter jsonMessageConverter() {
        return new Jackson2JsonMessageConverter();
    }
    
ja to bym widział poza klasą konfiguracji rabbita bo to zupełnie niezwiązane z tym rzeczy